vignettes/Life_Data_Analysis_Part_III.Rmd

```diff
@@ -104,9 +87,13 @@ mix_reg_plot
 ```
 
 ```{r segmented weibull II, fig.cap = "Figure 3: Subgroup-specific regression lines.", message = FALSE}
-# Using plot_mod_mix() to visualize regression lines of subgroups: 
-mix_reg_lines <- plot_mod_mix(mix_reg_plot, x = hours, status = state, 
-  mix_output = mixreg_weib, distribution = "weibull", title_trace = "Fitted Line")
+# Using plot_mod() to visualize regression lines of subgroups: 
+mix_reg_lines <- plot_mod(
+  mix_reg_plot, 
```

Comment:
Maybe this is not the right point to mention this ...

While I was documenting and checking mixmod_regression, I've often used the call 

plot_mod(plot_prob(_mixmod_regression_output_), _mixmod_regression_output_)

I asked myself whether this call is intuitive since the output of mixmod_regression is provided two times...plot_prob and again plot_mod

Reply:
I see your point. We could remove the argument `p_obj` from `plot_mod`. We should still keep the possibility to create just the probability plot with `plot_prob(mix_output)`

Reply:
And call `plot_prob` inside of `plot_mod`